scripts/is_python3_venv

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env python3
+import sys
+
+is_venv = (sys.prefix != getattr(sys, "base_prefix", sys.prefix))
```

Comment:
Bonus points if you add a reference to the documentation, which helps to understand why this works:
https://docs.python.org/3/library/venv.html#how-venvs-work

Reply:
I'm going to fix this (even if this PR is likely to be closed instead of being merged).